Cargo.toml

```diff
@@ -49,7 +49,7 @@ exclude = [
 ]
 
 [workspace.package]
-version = "0.14.0"
+version = "0.14.1"
 license = "MIT OR Apache-2.0"
 
 [workspace.dependencies]
```

Comment:
In case you missed them, also the internal deps down here will need to be dumped

Reply:
There was a script for updating these I think? Maybe the script needs a fix then? cc @tokatoka

Reply:
idk what i did is is open vscode and ctrl f and replace

no i'll just fix manually..